bookserver/db.py

```diff
@@ -52,6 +51,10 @@ async def init_models():
         await conn.run_sync(Base.metadata.create_all)
 
 
+async def term_models():
```

Comment:
Maybe some comments/documentation on what this is for?

Reply:
I'll add docs. I spent a long time finding out that failing to terminate the engine prevents PostgreSQL from truncating / dropping tables (used to reset the database to a clean state before each test).